src/js/components/Select/SelectContainer.js

```diff
@@ -207,6 +225,13 @@ const SelectContainer = forwardRef(
       [multiple, onChange, optionIndexesInValue, options, valueKey],
     );
 
+    const onClear = useCallback(
+      event => {
+        onChange(event, { option: '', value: '', selected: '' });
```

Comment:
I wonder if `option` should be `undefined` here?

Reply:
changes made